files/jenkins-slave.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+JENKINS_CONFIG=/etc/sysconfig/jenkins-slave
+
+# Read config
+[ -f "$JENKINS_CONFIG" ] && . "$JENKINS_CONFIG"
+
+if [ -x /sbin/runuser ] ; then
+    RUNUSER=runuser
+else
+    RUNUSER=su
```

Comment:
I don't think this wrapper script is needed at all since the user can be declared in the systemd service unit?